hw4/phonebook/phonebook.c

```diff
@@ -0,0 +1,265 @@
+﻿#define _CRT_SECURE_NO_WARNINGS
+
+#include <stdio.h>
+#include <time.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define ARRAY_SIZE 100
+#define RANGE 11
+
+typedef struct 
+{
+    char name[30];
+    char phone[30];
+} PhoneBookEntry;
+
+void findUser(char* inputName, PhoneBookEntry* phoneBook, int phoneBookSize, PhoneBookEntry* answer, int* counterUser)
```

Comment:
```suggestion
void findUser(const char* inputName, const PhoneBookEntry* phoneBook, int phoneBookSize, PhoneBookEntry* answer, int* counterUser)
```
Вообще, всё, что передаётся по указателю или как массив, надо отмечать const, если мы не собираемся менять их внутри функции. Так по сигнатуре функции сразу видно, кто входной параметр, кто выходной.